examples/units_and_coordinates/STEREO_SECCHI_starfield.py

```diff
@@ -24,23 +25,21 @@
 
 import sunpy.map
 from sunpy.coordinates import get_body_heliographic_stonyhurst
-from sunpy.net import helioviewer
+from sunpy.time import parse_time
 
 ###############################################################################
-# Let's download a STEREO-A SECCHI COR2 image from Helioviewer which provide
+# Let's download a STEREO-A SECCHI COR2 image from Helioviewer.org which provide
 # pre-processed images and load it into a Map.
 
-hv = helioviewer.HelioviewerClient()
-f = hv.download_jp2('2014/05/15 07:54', observatory='STEREO_A',
-                    instrument='SECCHI', detector='COR2')
-cor2 = sunpy.map.Map(f)
+cor2_file = hvpy.save_file(hvpy.getJP2Image(parse_time('2014/05/15 07:54').datetime, hvpy.DataSource.COR2_A.value), "COR2.JPEG2000")
+cor2_map = sunpy.map.Map(cor2_file)
```

Comment:
This is broken (see https://sunpy--6404.org.readthedocs.build/en/6404/generated/gallery/units_and_coordinates/STEREO_SECCHI_starfield.html#sphx-glr-generated-gallery-units-and-coordinates-stereo-secchi-starfield-py). I think `hvpy.save_file` doesn't return the filename, so you have to manually pass the filename into `Map`.

Reply:
This is going to be fixed in the next release of hvpy that will be out this week.

Reply:
Cool - lets wait for that, then add a pin to the docs requirements when the new version of `hvpy` is out